modules/release-notes/pages/sdk-release-notes.adoc

```diff
@@ -33,6 +33,16 @@ This {release} version of the software and programming language should not be us
 [[highlights]]
 == Highlights of what's new in {release}
 
+The {release} release only includes minor fixes and enhancements including the following:
+
+- The `+dfx start --clean+` command has been improved to no longer crash if you run the command in a project that is already in a clean state.
++
+Without this fix, manually removing the directories the command is intended to delete would result in an {IC} crash.
```

Comment:
```suggestion
```
It doesn't cause the IC to crash.

Reply:
what was the previous result before this fix? dfx exits without starting the replica?
the description from the PR says "dfx start --clean **crashes** on already clean project. "

Reply:
> dfx exits without starting the replica?

Yes. It doesn't crash the replica.